### PR TITLE
Checks for source repo in payload and adds as it a standard version property in graph.

### PR DESCRIPTION
**Pull request by GeetikaBatra**
Signed-off-by: Geetika Batra gbatra@redhat.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ HTTP Status Code: 200 success
 ```
 URL: /api/v1/selective_ingest
 ```
-Description: Import epv data and generate response for the POST request to /api/v1/selective. Its similar to ingest_to_graph, with a difference,
```
Comment (msrb) [inline README.md]: You can also update sample input for this API call as the payload has changed. The same for `/api/v1/ingest_to_graph`.

Comment (GeetikaBatra) [reply README.md]: @msrb Started working on it already :D. Will update the sample input for sure. Thanks :)

**Comment by GeetikaBatra** [comment]
[test]

**Comment by GeetikaBatra** [comment]
[test]

**Comment by msrb** [comment]
[test]

**Comment by msrb** [comment]
[test]

**Comment by GeetikaBatra** [comment]
Thanks @msrb. Merging it :)

**Pull request merged**